server/model/test_run_event.go

```diff
@@ -0,0 +1,49 @@
+package model
+
+import "time"
+
+type (
+	Protocol string
+	Status   string
+)
+
+var (
+	ProtocolHTTP Protocol = "http"
+	ProtocolGRPC Protocol = "grpc"
+)
+
+var (
+	StatusPassed  Status = "passed"
+	StatusWarning Status = "warning"
+	StatusFailed  Status = "failed"
+)
```

Comment:
Looks good, one minor question, does it make sense to move this to the `connection_test_result.go` model? Or do you think the `test_run_event` model is a better place for this?

Reply:
Initially, I add them in the `test_run_event`. I moved them from there because it's used by connection package and it doesn't have any relation to the `test_run_event`file. That's why they are not together.